package.json

```diff
@@ -21,12 +21,12 @@
     "db:studio": "prisma studio",
     "setup": "run-s db:generate db:push db:seed",
     "start": "cross-env NODE_ENV=production node ./build/server.js",
-    "start:mocks": "cross-env NODE_ENV=production node --require ./mocks --require dotenv/config ./build/server.js",
+    "start:mocks": "cross-env NODE_ENV=development node --require dotenv/config ./build/server.js dotenv_config_path=.env.example",
     "test": "vitest",
-    "test:e2e:dev": "start-server-and-test dev http://localhost:3000 \"npx cypress open\"",
+    "test:e2e:dev": "cross-env PORT=3000 start-server-and-test dev http://localhost:3000 \"playwright test\"",
```

Comment:
We'll have to make this `8080` so it works with what's on `main`, see https://github.com/Seneca-CDOT/starchart/commit/3417e4a47fe4eaee2650537e7eeed736c1c9e0a7#diff-1ba718c1eb8aa39cd20c2562d92523068c734d75f54655e97d652b992d9b4259R57